src/soulmates/index.scss

```diff
@@ -0,0 +1,4 @@
+@import '_core';
+@import '_adverts';
+@import '_adverts-soulmates';
+@import '_advert';
```

Comment:
I'm surprised you haven't seen more side effects. What happens if you put `_adverts-soulmates` last?